### PR TITLE
Added h - An html templating library written in kotlin

### PR DESCRIPTION
**Pull request by phenax**

### CODE REVIEW

File: _data/projects/H.yml [hunk 1]
```diff
@@ -0,0 +1,11 @@
...
+desc: Html templating library written in kotlin
+site: https://github.com/phenax/h
+tags:
+- html
+- kotlin
+- template
+- engine
+upforgrabs:
+  name: help wanted
+  link: https://github.com/phenax/h/issues/1
```
Comment (shiftkey) [inline _data/projects/H.yml]: This should be a link to a list of issues, rather than a single one, e.g: https://github.com/phenax/h/labels/help%20wanted

**Review by shiftkey — CHANGES_REQUESTED** [review]
All seems fine, just that one issue with the `link`

**Comment by phenax** [comment]
@shiftkey Whats the issue?

**Comment by phenax** [comment]
Oh. Sorry. The github client i was using doesn't show the change.

**Review by shiftkey — APPROVED** [review]

**Comment by shiftkey** [comment]
@phenax thanks!

**Pull request merged**